gossip/src/crds_value.rs

```diff
@@ -494,27 +499,77 @@ impl Sanitize for NodeInstance {
 pub struct RestartLastVotedForkSlots {
     pub from: Pubkey,
     pub wallclock: u64,
-    pub slots: Vec<CompressedSlots>,
+    pub compressed_slots: Vec<u8>,
+    pub uncompressed_bytes: u64,
+    pub last_voted_slot: Slot,
     pub last_voted_hash: Hash,
     pub shred_version: u16,
 }
 
 impl Sanitize for RestartLastVotedForkSlots {
     fn sanitize(&self) -> std::result::Result<(), SanitizeError> {
-        if self.slots.is_empty() {
+        if self.last_voted_slot == 0 {
             return Err(SanitizeError::InvalidValue);
         }
-        self.slots.sanitize()?;
         self.last_voted_hash.sanitize()
     }
 }
 
 impl RestartLastVotedForkSlots {
-    pub fn new(from: Pubkey, now: u64, last_voted_hash: Hash, shred_version: u16) -> Self {
+    pub fn new(
+        from: Pubkey,
+        now: u64,
+        last_voted_fork: &mut [Slot],
+        last_voted_hash: Hash,
+        shred_version: u16,
+    ) -> Self {
+        let last_voted_slot;
+        let mut compressed_slots;
+        let uncompressed_bytes;
+        if last_voted_fork.is_empty() {
+            last_voted_slot = 0;
+            compressed_slots = Vec::with_capacity(1);
+            uncompressed_bytes = 1;
+        } else {
+            last_voted_fork.sort();
+            last_voted_fork.reverse();
```

Comment:
`sort_by_key` + `Revese`, as in:
https://github.com/solana-labs/solana/blob/f13c78b7c/core/src/repair/serve_repair.rs#L686

Reply:
Done.